src/renderer/components/+workloads-pods/details/volumes/variants/projected.tsx

```diff
@@ -25,7 +25,11 @@ export const Projected: VolumeVariantComponent<"projected"> = (
                   </DrawerItem>
                   <DrawerItem name="Items">
                     <ul>
-                      {secret.items?.map(({ key }) => <li key={key}>{key}</li>)}
+                      {secret.items?.map(({ key, path }) => (
+                        <li key={key}>
+                          {key} ⇢ {path}
```

Comment:
Is the arrow appropriate? Use a colon?

Reply:
It is a "mapping" so I think either should be fine.

Reply:
as long as we're consistent